### PR TITLE
Fix the Client setup flag

### PR DESCRIPTION
**Pull request by pragamj**
Signed-off-by: pragam <pragam.jain@seagate.com>

# Problem Statement
- Fix Client_setup_flag

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [x] Collection tested and no collection error introduced (`pytest --local True --collect-only`)
[collections_log.txt](https://github.com/Seagate/cortx-test/files/8683822/collections_log.txt)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Review by priyankaborawake — APPROVED** [review]

File: libs/prov/prov_k8s_cortx_deploy.py [hunk 1]
```diff
@@ -1293,53 +1293,53 @@ def test_deployment(self, sns_data, sns_parity,
...
+        if self.deployment_type not in self.exclusive_pod_list:
+            if setup_client_config_flag:
+                LOGGER.info("Setting the current namespace")
+                resp_ns = master_node_list[0].execute_cmd(
+                    cmd=common_cmd.KUBECTL_SET_CONTEXT.format(namespace),
+                    read_lines=True)
+                LOGGER.debug("response is %s,", resp_ns)
+                resp = system_utils.execute_cmd(
+                    common_cmd.CMD_GET_IP_IFACE.format(self.deploy_cfg['iface']))
+                eth1_ip = resp[1].strip("'\\n'b'")
```
Comment (pranav-sahasrabudhe) [inline libs/prov/prov_k8s_cortx_deploy.py:1305]: Is this a specific thing to name it eth1_ip? or can it be more generic?

File: libs/prov/prov_k8s_cortx_deploy.py [hunk 2]
```diff
@@ -1293,53 +1293,53 @@ def test_deployment(self, sns_data, sns_parity,
...
+                    LOGGER.debug("External LB value, ip and port will be: %s", ext_port_ip)
+                else:
+                    LOGGER.info("Configure HAproxy on client")
+                    ext_lbconfig_utils.configure_haproxy_rgwlb(master_node_list[0].hostname,
+                                                               master_node_list[0].username,
+                                                               master_node_list[0].password,
+                                                               eth1_ip, self.deploy_cfg['iface'])
+                    ext_port_ip = self.deploy_cfg['https_protocol'].format(eth1_ip)
+                LOGGER.info("Step to Create S3 account and configure credentials")
+                if self.s3_engine == 2:
```
Comment (pranav-sahasrabudhe) [inline libs/prov/prov_k8s_cortx_deploy.py:1324]: What is the meaning of value "2" here? Some comment would help.

**Review by pranav-sahasrabudhe — CHANGES_REQUESTED** [review]
Please take a look at the comments and kindly don't resolve th

File: libs/prov/prov_k8s_cortx_deploy.py (hunk 1 above)
Comment (pragamj) [reply libs/prov/prov_k8s_cortx_deploy.py:1305]: its being specific to pick first interface so just a variable name we thought of using eth1_ip

File: libs/prov/prov_k8s_cortx_deploy.py (hunk 2 above)
Comment (pragamj) [reply libs/prov/prov_k8s_cortx_deploy.py:1324]: 2 is denoted for rgw based operations its being coming from setup entry and we are comparing it

File: libs/prov/prov_k8s_cortx_deploy.py (hunk 1 above)
Comment (pranav-sahasrabudhe) [reply libs/prov/prov_k8s_cortx_deploy.py:1305]: OK. I am fine as far as it's as per the requirement.

File: libs/prov/prov_k8s_cortx_deploy.py (hunk 2 above)
Comment (pranav-sahasrabudhe) [reply libs/prov/prov_k8s_cortx_deploy.py:1324]: OK. So kindly add an inline comment as appropriate. Thanks!

**Review by pranav-sahasrabudhe — APPROVED** [review]
Approved

**Review by ketanarlulkar — APPROVED** [review]

**Review by ravindrachoudhari — APPROVED** [review]

**Pull request merged**